app/app.go

```diff
@@ -113,7 +128,6 @@ import (
 	tendermintdb "github.com/tendermint/tm-db"
 
 	appparams "github.com/persistenceOne/persistenceCore/v3/app/params"
```

Comment:
we need to upgrade all v3 imports to v4 as well no?

Reply:
oo yea, will do